scripts/docker-entrypoint.sh

```diff
@@ -279,9 +279,17 @@ unset DB_USERNAME DB_PASSWORD SECRET_DB_USERNAME SECRET_DB_PASSWORD CASSANDRA_US
 
 unset pega_root lib_root config_root
 
+
 # Run tomcat if the first argument is run otherwise try to run whatever the argument is a command
-if [ "$1" = 'run' ]; then
-  exec catalina.sh "$@"
-else
-  exec "$@"
+if [ ! -z "$1" ]; then
+  sudo_args=""
+  if [ "$RUN_AS_RESTRICTED_USER" == "true" ]; then 
+    sudo_args="sudo -n -E -u tomcat " 
```

Comment:
@khick77 Do we want to document the significance of tomcat in [README ](RUN_AS_RESTRICTED_USER)as we are exposing new environment variable **RUN_AS_RESTRICTED_USER**?